### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by IvanGrginovInf**
Fixes broken tests as part of https://github.com/infinum/eightshift-libs/pull/211

Changelog
---
- `checkAttr` tests
- `props` tests (done less detailed than the ones in JS since that was a bit of on overkill + the props helper is much more simplified now)
- Removed tests which test native PHP functionality (and cause IntelliSense errors in editor)
- Fixed some IntelliSense errors / typos

### CODE REVIEW

File: tests/Helpers/ComponentHelpersTest.php [hunk 1]
```diff
@@ -230,7 +218,20 @@ function ($argument) {
...
+	$manifest = Components::getManifest(dirname(__FILE__, 2) . '/data/src/Blocks/components/button/');
+	$attributes = [
+		'prefix' => 'prefixedMultipleTimesButton',
+		'prefixedMultipleTimesButtonAlign' => 'right'
+	];
+
+	$results = Components::checkAttr('buttonAlign', $attributes, $manifest);
+
+	$this->assertIsString($results, 'Result should be a string');
+	$this->assertEquals('right', $results, "The set attribute should be {$attributes['prefixedMultipleTimesButtonAlign']}");
```
Comment (dingo-d) [inline tests/Helpers/ComponentHelpersTest.php:233]:
```suggestion
	$this->assertSame('right', $results, "The set attribute should be {$attributes['prefixedMultipleTimesButtonAlign']}");
```

Always use strict check (type and value). `assertEquals` will just check the value.

File: tests/Helpers/ComponentHelpersTest.php [hunk 2]
```diff
@@ -1033,11 +1033,69 @@ function ($argument) {
...
+		->shouldReceive('getProjectPath')
+		->andReturn('tests/data');
+
+	$this->blocksExample = new BlocksExample();
+	$this->blocksExample->getBlocksDataFullRaw();
+	$output = Components::props($attributes, 'heading');
+
+	$this->assertIsArray($output);
+	$this->assertContains('prefix', array_keys($output), "Output array doesn't contain prefix attribute key.");
+	$this->assertEquals('heading', $output['prefix']);
```
Comment (dingo-d) [inline tests/Helpers/ComponentHelpersTest.php:1064]:
```suggestion
	$this->assertSame('heading', $output['prefix']);
```

File: tests/Helpers/ComponentHelpersTest.php [hunk 3]
```diff
@@ -1033,11 +1033,69 @@ function ($argument) {
...
+
+	// Next level
+	$output = Components::props($output, 'typography');
+	$this->assertIsArray($output);
+	$this->assertContains('prefix', array_keys($output), "Output array doesn't contain prefix attribute key.");
+	$this->assertEquals('headingTypography', $output['prefix']);
```
Comment (dingo-d) [inline tests/Helpers/ComponentHelpersTest.php:1070]:
```suggestion
	$this->assertSame('headingTypography', $output['prefix']);
```

File: tests/Helpers/ComponentHelpersTest.php (hunk 1 above)
Comment (IvanGrginovInf) [reply tests/Helpers/ComponentHelpersTest.php:233]: Got `109` results for `assertEquals` in our tests. 😄  but will change it.

**Review by dingo-d — CHANGES_REQUESTED** [review]
Left some minor comments 👍🏼

File: tests/Helpers/ComponentHelpersTest.php (hunk 1 above)
Comment (dingo-d) [reply tests/Helpers/ComponentHelpersTest.php:233]: 😱

**Comment by IvanGrginovInf** [comment]
> Left some minor comments 👍🏼

Replaced all instances of `assertEquals` with `assertSame`, all tests still pass. 🎉

**Comment by IvanGrginovInf** [comment]
@iruzevic please check the changes in `src/Helpers/Components.php`. This was causing tests to fail (`$attributeValue` being set to array as a fallback in `setVariablesToBreakpoints()` but then that value was used in a `str_replace()` in `variablesInner()`) but I'm not sure if the change properly fixes the issue. The tests pass with that change tho.

**Review by dingo-d — APPROVED** [review]

**Comment by dingo-d** [comment]
Tip to other reviewers: go commit by commit 🙂 Last one is just changing `assertEquals` to `assertSame` 🙂

**Review by goranalkovic-infinum — APPROVED** [review]

**Pull request closed without merging**